engine/src/main/java/org/terasology/rendering/world/viewDistance/ViewDistanceChangedEvent.java

```diff
@@ -22,7 +22,7 @@
 /**
  * Notifies the server that the clients view distance changed.
  *
- * If you want to change the view distance send {@link ViewDistanceChangeRequest} to the local player's client entity.
+ * If you want to change the view distance send {@link ViewDistanceChangedEvent} to the local player's client entity.
```

Comment:
I'm pretty sure that this claim is not true. The particular commit responsible for removal of ViewDistanceChangeRequest can be seen at https://github.com/MovingBlocks/Terasology/commit/66ec2c96a1ba7cff79e71eba7cf24b3bb1ded2cb#diff-a4fba91c13c6b1e0e34060570701a54a, and from the looks of it, it doesn't look like the behaviour was reimplemented.

Reply:
If I understand your comment correctly, you are saying that the claim that ViewDistanceChangeRequest is required to be sent to change the view distance is not true because it was deleted as seen in the attached diff, and never re-implemented. (and my change is valid :P)

Or do you mean that other than the fact ViewDistanceChangeRequest was deleted and never re-implemented, the claim that ViewDistanceChangedEvent is needed to change the view distance is false. (although I can see ViewDistanceChangedEvent being used in ServerViewDistance, I am not familiar with the functionality)

Thanks for the feedback though.

Reply:
The latter. ViewDistanceChangedEvent is sent only from client to server to inform the server of client changing the view distance, but as far as I can tell, there is no way to comunicate something like that the other way.

Reply:
Hmm, makes sense. Please check the change and let me know if that reflects the functionality.